code/messaging/src/main/java/com/adobe/marketing/mobile/aepcomposeui/aepui/state/SmallImageUIState.kt

```diff
@@ -0,0 +1,20 @@
+/*
+  Copyright 2024 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.aepcomposeui.aepui.state
+
+/**
+ * Data class representing the state of a small image UI.
+ *
+ * @property title The title of the small image UI.
```

Comment:
`title` is not a changeable state. Let's remove it in a future PR when the call is going to be implemented fully